Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -2475,7 +2475,11 @@ def normalize_scan_data(scan_data: dict) -> dict:
         include_none=True,
     )
 
-    result["TotalTime"] = readable_duration_time(scan_data["duration"])
+    if "duration" in scan_data:
+        result["TotalTime"] = readable_duration_time(scan_data["duration"])
```

Comment:
If duration is present but is empty, this will also raise an error. This should cover both cases.
```suggestion
    if scan_data.get("duration"):
        result["TotalTime"] = readable_duration_time(scan_data["duration"])
```